src/main/java/ladder/view/ResultView.java

```diff
@@ -44,10 +49,37 @@ private String printUsedOrNot(Point point) {
   }
 
 
+  private String printPrize(Prizes prizes) {
+    return prizes.prizes()
+        .map(prize -> prize.prize())
+        .map(prize -> String.format(NAMES_FORMAT, prize))
+        .collect(Collectors.joining());
+  }
+
   private String printName(Users users) {
-    return users.getUsers().stream()
-        .map(user -> user.getName())
+    return users.users()
+        .map(user -> user.name())
         .map(name -> String.format(NAMES_FORMAT, name))
         .collect(Collectors.joining());
   }
+
+  public void printAllResult(PrizeResult prizeResult) {
+    prizeResult.result()
+        .entrySet()
+        .forEach(result -> System.out.println(
+            result.getKey().name() + " : " + result.getValue().prize()));
```

Comment:
```suggestion
    prizeResult.result()
            .forEach((key, value) -> System.out.println(
                    key.name() + " : " + value.prize()));
```
entrySet()으로 변경없이 바로 forEach를 호출할 수 있어요. 🙂